tutorials/cyber/network_mapping/lanl_network_mapping_using_rapids.ipynb

```diff
@@ -0,0 +1,801 @@
+    "    regex_dict = {}\n",
+    "    for f in os.listdir(yaml_directory):\n",
+    "        temp_regex = load_regex_yaml(yaml_directory + '/' + f)\n",
+    "        regex_dict[f[:-5]] = temp_regex\n",
+    "        \n",
+    "    return regex_dict\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run Parsing Data Pipeline\n",
+    "\n",
+    "In this instance, we'll be focusing on parsing raw Windows Event Logs that are of event code [4624](https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=4624) and [4625](https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=4625)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir -p data/input/lanl\n",
+    "!if [ ! -f data/input/lanl/wls.csv ]; then tar -xzvf /data/lanl/wls.tar.gz -C data/input/lanl; fi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(2515, 32)\n"
+     ]
+    }
+   ],
+   "source": [
+    "#raw lanl data parsing.\n",
+    "AUTH_INPUT_PATH = 'data/input/lanl/wls.csv'\n",
+    "REGEX_CONF_PATH = 'conf/lanl_regex_configs'\n",
+    "EVENT_CODES_OF_INTEREST = ['4624','4625']\n",
+    "REQUIRED_COLS = ['Time','EventID','Raw']\n",
+    "DELIMITER = ','\n",
+    "\n",
+    "logs_gddf = dask_cudf.read_csv(AUTH_INPUT_PATH,\n",
+    "                               names=INPUT_COLS_SET,\n",
+    "                               delimiter=DELIMITER,\n",
+    "                               usecols=REQUIRED_COLS,\n",
+    "                               dtype=INPUT_DTYPES,\n",
+    "                               skip_blank_lines=True,\n",
+    "                              )\n",
+    "    \n",
+    "REGEX_MAPPINGS = create_regex_dictionaries(REGEX_CONF_PATH)\n",
+    "\n",
+    "parts = [dask.delayed(pipeline)(x, EVENT_CODES_OF_INTEREST, REGEX_MAPPINGS) for x in logs_gddf.to_delayed()]\n",
+    "temp_df = dask_cudf.from_delayed(parts)\n",
+    "# Bring data back to a single GPU, for downstream graph analytics\n",
+    "auth_gdf = temp_df.compute()\n",
+    "print(auth_gdf.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read edge definitions from JSON file\n",
+    "\n",
+    "Now the parsing of Windows Event Logs has concluded, next we prepare for the network mapping portion. Within the edge definitions configuration file we define our edges by indicating the source and destination for each edge, referencing the column names of our input data.  \n",
+    "  \n",
+    "Below we also read in the netflow data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = 'conf/network_mapping_configs/edge-definitions.json'\n",
```

Comment:
This filename and folder structure does not match the existing file in the repository.

Reply:
Thanks, I'll take a look

Reply:
I have updated this. Thank you